src/tests/Interop/NativeLibrary/API/NativeLibraryTests.csproj

```diff
@@ -26,8 +26,18 @@
     <ItemGroup>
       <AssembliesToCopy Include="$(OutDir)/libNativeLibrary.*" />
       <AssembliesToCopy Include="$(OutDir)/NativeLibrary.*" />
-      <AssembliesToCopy Include="$(OutDir)/$(TargetName).dll" />
     </ItemGroup>
     <Copy SourceFiles="@(AssembliesToCopy)" DestinationFiles="@(AssembliesToCopy -> '$(NativeLibrarySubdirectory)/%(Filename)$(FileNameSuffix)%(Extension)')" />
   </Target>
+
+  <Target Name="SetUpSubdirectoryManaged" AfterTargets="Build" Condition="'$(__SkipManaged)' != '1'">
+    <PropertyGroup>
+      <ManagedLibrarySubdirectory>$(OutDir)/subdirectory</ManagedLibrarySubdirectory>
+      <FileNameSuffix>-in-subdirectory</FileNameSuffix>
```

Comment:
Can we pull these out so that they are shared between the two `SetUpSubdirectory*` targets?

Reply:
Yes, I pulled out these properties and shared it between targets